frontend/src/Components/Results/Results.tsx

```diff
@@ -76,6 +76,26 @@ function ResultDisplay(props: ResultDisplayProps) {
                 .toString()}
             />
           </div>
+          <div className="col-lg-6">
+            <ReportCard
+              description="Average agent age upon death"
+              title={Average(result.deaths.map((d) => d.ageUponDeath))
+                .toFixed(3)
+                .toString()}
+            />
+          </div>
+          <div className="col-lg-6">
+            <ReportCard
+              description="Max agent age upon death"
+              title={Max(result.deaths.map((d) => d.ageUponDeath)).toString()}
+            />
+          </div>
+          <div className="col-lg-6">
+            <ReportCard
+              description="Min agent age upon death"
+              title={Min(result.deaths.map((d) => d.ageUponDeath)).toString()}
+            />
+          </div>
```

Comment:
Since this is getting larger could we move it to a separate component `Stats.tsx` which takes in results in the props? Not necessary feel free to ignore and resolve the comment

Reply:
Yeah for sure can do